src/Microsoft.Extensions.Logging.TraceSource/TraceSourceScope.cs

```diff
@@ -14,11 +14,13 @@ internal class TraceSourceScope : IDisposable
         // To detect redundant calls
         private bool _isDisposed;
 
+#if NET451
```

Comment:
Hmmm, should still have some sort of doc comments for non-net451. Ditto elsewhere